packages/liveblocks-node/src/client.ts

```diff
@@ -125,6 +137,7 @@ export type RoomData = {
   usersAccesses: RoomAccesses;
   groupsAccesses: RoomAccesses;
   metadata: RoomMetadata;
+  experimental_textEditor?: RoomTextEditor;
```

Comment:
Just curious what this is, and the reason it's at the room level. 

Does our system allow for a room to have both lexical / tiptap?

In which case, should this be something that is stored somewhere else than the room (but with a relation to the room)

Really coming without knowledge so feel free to point me to documentation or anything 😄

Reply:
Hey @FlowFlorent 👋🏻 

The property [`textEditor`](https://github.com/liveblocks/liveblocks-backend/blob/074d088db168ee44832a7237d04316241c10267e/apps/cloudflare/src/mongo/types.ts) is on the `MongoRoom` DB object since 5 month now from what I see. 
As you can see it's made to tell if a room as an associated text editor (lexical or tiptap but never both) or not. I think you should ask @jrowny if you want more info about why it's at the root level 🙂 
You can see the text editor is reported in [LiveblocksPlugin](https://github.com/liveblocks/liveblocks/blob/b1bc5342fafebcc5da7a225786818614681be09c/packages/liveblocks-react-lexical/src/liveblocks-plugin-provider.tsx#L180) for Lexical and in [useLiveblocksExtension](https://github.com/liveblocks/liveblocks/blob/b1bc5342fafebcc5da7a225786818614681be09c/packages/liveblocks-react-tiptap/src/LiveblocksExtension.ts#L89) for Tiptap

In my case I have added on the node client the property `experimental_textEditor` because a few days ago I found that I needed this property from the mongo db object which wasn't formatted in [formatRoom](https://github.com/liveblocks/liveblocks-backend/blob/fe4e9c92c8784cb9886bc7090280567c8db4b79d/apps/cloudflare/src/core/formatters.ts). 
But as it's for more internal cooking (e.g only the dashboard and the package `@liveblocks/emails`) I preferred to mark this property `experimental_` to be free to do whatever w/ it and break it if needed.

Reply:
This is not meant to be seen by end users. Instead of experimental, can you just use the @internal annotation like we do for `baseUrl`?

Reply:
Let me check

Reply:
Thanks for pointing this out @jrowny 🙏🏻 
I made the property more "hidden" in this [commit](https://github.com/liveblocks/liveblocks/pull/2042/commits/586ccbe2026a668fac00de1984bbbcf38978dcb9). 

For now we will keep the `experimental_` part as it's coming from the back-end. I will later to rename it if needed.